pkg/resource/validations/kms_arn_regex_validation_test.go

```diff
@@ -41,6 +41,17 @@ var _ = Describe("Validations", func() {
 			})
 		})
 
+		Context("when kmsKeyARN is not empty but is not prefixed with 'mrk'", func() {
```

Comment:
We we should use the Context for grouping tests together, right here we are just setting the context for a single test.